taxi/tests/test_admin.py

```diff
@@ -0,0 +1,34 @@
+from django.urls import reverse
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+
+
+class AdminTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="Test_admin",
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="user",
+            password="testdriver",
+            license_number="123456",
+        )
```

Comment:
The `create_user` method is missing the `email` parameter, which might be required if the user model mandates it. Ensure that all required fields for user creation are provided.